api/src/lib/helpers.ts

```diff
@@ -202,6 +202,16 @@ export const abbreviateUserName = <T extends { firstName: string; lastName: stri
     return user.firstName;
 };
 
+export const getUserFullName = <T extends { firstName: string; lastName: string | null } | undefined | null>(
+    user: T
+): string => {
+    if (!user) {
+        return 'Anonymous User';
+    }
+
+    return user.firstName + (user.lastName ? ' ' + user.lastName : '');
```

Comment:
I spent way too long looking into a nicer alternative to this. 

Came up with this:
`return [user.firstName, user.lastName].filter(n => !!n).join(' ')`

I don't think that's any better but I spent time figuring this out so thought I'd share it :joy:

Reply:
I can see why you looked into it because my line is pretty hard to read for such a simple idea. Perhaps it would be clearest over multiple lines:

`
if (user.lastName) {
    return user.firstName + ' ' + user.lastName;
} else {
    return user.firstName;
}
`

Not sure if that is any better either!

Reply:
Use interpolated string and then yeah I think it looks a lot better: `if (user.lastName) { return `${user.firstName} ${user.lastName}`; } else { return user.firstName; }`

Reply:
Ah github stripped the ` bits, you get the gist though

Reply:
Nice. Made this change now.